### ISSUE TITLE
[TW-415] Parent project should be listed in 'projects' report, even if all tasks associated with child projects

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#450]
_Florian Hollerweger on 2012-09-13T08:33:10Z says:_ 

 If a parent project does not have any tasks associated with it (because all tasks for the respective project are associated with child projects), it is not listed in the 'projects' report. This results in hard-to-read output - consider the first clumn of an imaginary 'projects' report. 

home
  home.shopping
  home.car
work
  work.project1
  work.project2
  tax.2011
  tax.2012

For readability, I propose to introduce a line for the parent 'tax' project, even though it does not have any tasks associated with it (which would be made explicit through the following columns of the 'projects' report anyway).

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2012-09-13T08:33:10Z
Modified: 2014-02-09T01:53:43Z
```

**Comment by taskwarrior** [comment]
_Florian Hollerweger on 2012-09-13T08:34:46Z says:_ 

 Again the output of that imaginary report, this time hopefully with formatting, to illustrate my point even more clearly:

  home
    home.shopping
    home.car
  work
    work.project1
    work.project2
    tax.2011
    tax.2012

**Comment by taskwarrior** [comment]
_Aikido Guy on 2012-09-13T14:21:08Z says:_ 

 I second this suggestion :)

http://taskwarrior.org/boards/1/topics/2439

**Comment by taskwarrior** [comment]
_Dirk Deimeke on 2012-09-13T18:02:17Z says:_ 

 This seems to be a duplicate of #1056

If you agree, please close the ticket.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2012-09-24T21:57:20Z says:_ 

 This should be fixed now in the newest 2.2.0 branch.

Testing is welcome! Please report any problems.
